pymc3/model.py

```diff
@@ -1055,6 +1100,43 @@ def _repr_latex_(self, name=None, dist=None):
     __latex__ = _repr_latex_
 
 
+def set_data(new_data, model=None):
+    """Sets the value of one or more data container variables.
+
+    Parameters
+    ----------
+    new_data : dict
+        New values for the data containers. The keys of the dictionary are
+        the  variables names in the model and the values are the objects
+        with which to update.
+
+    Examples
+    --------
+
+    .. code:: ipython
+
+        >>> import pymc3 as pm
+        >>> with pm.Model() as model:
+        ...    x = pm.Data('x', [1., 2., 3.])
+        ...    y = pm.Data('y', [1., 2., 3.])
+        ...    beta = pm.Normal('beta', 0, 1)
+        ...    obs = pm.Normal('obs', x * beta, 1, observed=y)
+        ...    trace = pm.sample(1000, tune=1000)
+
+    Set the value of `x` to predict on new data.
+
+    .. code:: ipython
+        >>> pm.set_data(new_data={'x': [5,6,9]}, model=model)
+        >>> # Another way to do the same thing using the model context is:
```

Comment:
I would say "using model itself"

Reply:
Minor english correction, @ferrine:

`using the model itself`